src/openrct2-ui/windows/Guest.cpp

```diff
@@ -1839,27 +1839,27 @@ void window_guest_inventory_update(rct_window* w)
     }
 }
 
-static std::pair<rct_string_id, Formatter> window_guest_inventory_format_item(Peep* peep, int32_t item)
+static std::pair<rct_string_id, Formatter> window_guest_inventory_format_item(Peep* peep, ShopItem item)
 {
     auto& park = OpenRCT2::GetContext()->GetGameState()->GetPark();
     auto parkName = park.Name.c_str();
 
     // Default arguments
     auto ft = Formatter();
-    ft.Add<uint32_t>(ShopItems[item].Image);
-    ft.Add<rct_string_id>(ShopItems[item].Naming.Display);
+    ft.Add<uint32_t>(ShopItems[static_cast<uint8_t>(item)].Image);
```

Comment:
```suggestion
    ft.Add<uint32_t>(ShopItems[EnumValue(item)].Image);
```
Also applies to other occurrences.